docs/en/stack/ml/index-custom-title-page.html

```diff
@@ -0,0 +1,218 @@
+<style>
+    * {
+      box-sizing: border-box;
+    }
+  
+    .card {
+      cursor: pointer;
+      padding: 16px;
+      text-align: left;
+      color: #000;
+    }
+  
+    .card:hover {
+      box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+      padding: 16px;
+      text-align: left;
+    }
+  
+    #guide a.no-text-decoration:hover {
+      text-decoration: none!important;
+    }
+  
+    .icon {
+      width: 24px;
+      height: 24px;
+      background-position: bottom;
+      background-size: contain;
+      background-repeat: no-repeat;
+    }
+  
+    .ul-col-1 {
+      columns: 1;
+      -webkit-columns: 1;
+      -moz-columns: 1;
+    }
+  
+    @media (min-width:769px) {
+      .ul-col-md-2 {
+        columns: 2;
+        -webkit-columns: 2;
+        -moz-columns: 2;
+      }
+    }
+
+    #guide h3.gtk {
+    margin-top: 0;
+  }
+
+  .mb-4, .my-4 {
+    margin-bottom: 0!important;
+  }
+  </style>
+  
+  <div class="legalnotice"></div>
+  
+  <div class="row my-4">
+    <div class="col-md-6 col-12">
+      <p></p>
+      <h2>Analyze your data with Elastic machine learning</h2>
+      <p>
+        Find anomalies and outliers, forecast based on trends, analyze natural language data, and identify areas of interest in your data with machine learning features.
+      </p>
+      <p>
+        <a href="https://www.elastic.co/guide/en/machine-learning/current/machine-learning-intro.html">
+          <button class="btn btn-primary">Get started</button>
+        </a>
+      </p>
+      <p>
+        <a class="inline-block mr-3" href="setup.html">Setup and security</a>
+    </p>
+    </div>
+    <div class="col-md-6 col-12">
+      <img class="w-100" src="https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/bltdd4cccf507615a8b/63e62da759036b11199d0f6b/ML_cover.jpg" />
+    </div>
+  </div>
+  
+  <h3 class="gtk">Get to know Elastic machine learning</h3>
```

Comment:
This section could do with more padding above it I think. It seems too close to the 'Setup and security' link.

Reply:
Fixed in [31dbd38](https://github.com/elastic/stack-docs/pull/2326/commits/31dbd382e6971a9bf1c4296c718960d364fbea1a).